data/avgi/avgi jobs.txt

```diff
@@ -0,0 +1,348 @@
+	npc
+		government "Avgi (Dissonance)"
+		personality coward harvests timid secretive surveillance
+			confusion 20
+		system
+			distance 1 2
+		fleet "Small Dissonance Merchants"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 30000
+		dialog phrase "avgi nuclear materials"
+
+mission "Avgi: Special Nuclear Material Shipment 3"
+	job
+	repeat
+	name "Special nuclear materials to <planet>"
+	description "Deliver <cargo> special nuclear material to <planet> by <date> in exchange for <payment>. Be on the lookout for pirates looking to steal the nuclear material for nefarious purposes."
+	deadline
+	cargo "Avgi Radioisotopes" 3 6 .3
+	to offer
+		has "Avgi: Nuclear Material Transport Jobs"
+		not "Avgi: Fissiles Shortage"
+		random < 6
+	source
+		government "Avgi (Twilight Guard)"
+	destination
+		government "Avgi (Twilight Guard)"
+	npc
+		government "Avgi (Dissonance Angry)"
+		personality coward nemesis harvests plunders
+			confusion 20
+		system
+			distance 1 2
+		fleet "Large Dissonance Pirates"
+	npc
+		government "Avgi (Dissonance)"
+		personality coward harvests timid secretive surveillance
+			confusion 20
+		system
+			distance 1 2
+		fleet "Small Dissonance Merchants"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 30000
+		dialog phrase "avgi nuclear materials"
+
+
+phrase "avgi infocourier delivery"
+	word
+		`You drop off the `
+	word
+		`bricks`
+		`slabs`
+		`rods`
+		`tiles`
+	word
+		` of `
+	word
+		`<commodity> at a designated terminal and receive <payment> in payment.`
+
+mission "Avgi: Information Courier"
+	job
+	repeat
+	name "Information courier to <planet>"
+	description "Bring <cargo> storage media to update <planet>'s local network by <date> in exchange for <payment>."
+	deadline
+	cargo "Avgi Storage Media" 3 6 .3
+	to offer
+		has "Avgi: Infocourier Jobs"
+		random < 15
+	source
+		government "Avgi (Consonance)" "Avgi (Twilight Guard)"
+	destination
+		government "Avgi (Consonance)" "Avgi (Dissonance)" "Avgi (Twilight Guard)"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 12000 300
+		dialog phrase "avgi infocourier delivery"
+
+mission "Avgi: Information Courier Rush"
+	job
+	repeat
+	name "Information courier to <planet>"
+	description "Bring <cargo> storage media to update <planet>'s local network by <date> in exchange for <payment>."
+	deadline 1 1
+	cargo "Avgi Storage Media" 3 6 .3
+	to offer
+		has "Avgi: Infocourier Jobs"
+		random < 15
+	source
+		government "Avgi (Consonance)" "Avgi (Twilight Guard)"
+	destination
+		government "Avgi (Consonance)" "Avgi (Dissonance)" "Avgi (Twilight Guard)"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 12000 300
+		dialog phrase "avgi infocourier delivery"
+
+
+phrase "avgi twilight guards dropoff"
+	word
+		`You say your goodbyes to the Twilight Guardsmen and receive <payment> in payment.`
+
+mission "Avgi: Twilight Guardsmen Transport"
```

Comment:
For some reason the only destination this mission can have is Feo Platform, not sure why.